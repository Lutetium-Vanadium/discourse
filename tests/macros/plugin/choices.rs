fn main() {
    requestty::questions![Plugin { choices: todo!() }];
}
