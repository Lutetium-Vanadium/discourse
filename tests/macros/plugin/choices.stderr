error: option `choices` does not exist for kind `plugin`
 --> $DIR/choices.rs:2:36
  |
2 |     requestty::questions![Plugin { choices: todo!() }];
  |                                    ^^^^^^^
